tools/gengen/util/generator.go

```diff
@@ -334,9 +334,12 @@ func (g *GenesisGenerator) setupMiners(ctx context.Context) ([]*RenderedMinerInf
 		}
 
 		// Add configured deals to the market actor with miner as provider and worker as client
-		dealIDs, err := g.commitDeals(ctx, wAddr, mIDAddr, m)
-		if err != nil {
-			return nil, err
+		dealIDs := []abi.DealID{}
+		if len(m.CommittedSectors) > 0 {
```

Comment:
I forget why this blows up if the miner has no committed sectors, but clearly we don't need to commitDeals if we have no sectors.